modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/services/syncml/impl/SyncmlServiceImpl.java

```diff
@@ -0,0 +1,285 @@
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.wso2.carbon.mdm.mobileservices.windows.services.syncml.impl;
+
+import org.wso2.carbon.mdm.mobileservices.windows.common.exceptions.FileOperationException;
+import org.wso2.carbon.mdm.mobileservices.windows.services.syncml.SyncmlService;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.w3c.dom.Document;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.wso2.carbon.device.mgt.common.Device;
+import org.wso2.carbon.device.mgt.common.DeviceManagementConstants;
+import org.wso2.carbon.device.mgt.common.DeviceManagementException;
+import org.wso2.carbon.device.mgt.common.DeviceManagementServiceException;
+import org.wso2.carbon.mdm.mobileservices.windows.services.syncml.util.SyncmlUtils;
+import javax.ws.rs.core.Response;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.wso2.carbon.mdm.mobileservices.windows.common.Constants;
+
+/**
+ * Implementing class of SyncmlImpl interface.
+ */
+public class SyncmlServiceImpl implements SyncmlService {
+
+	private static final String MESSAGE_ID_ONE_TEMP = "1";
+	private static final String MESSAGE_ID_TWO_TEMP = "2";
+	private static final int FIRST_ITEM = 0;
+	private static final int SECOND_ITEM = 1;
+	private static final String OS_VERSION = "osVersion";
+	private static final String IMSI = "imsi";
+	private static final String IMEI = "imei";
+	private static final String VENDOR = "vendor";
+	private static final String MODEL = "model";
+	private enum DevicePropertyIndex {
+		OS_VERSION(3),
+		IMSI(4),
+		IMEI(5),
+		DEVICE_ID(6),
+		DEVICE_MANUFACTURER(7),
+		DEVICE_MODEL(8),
+		DEVICE_LANGUAGE(9);
+		private final int itemPosition;
+		private DevicePropertyIndex(final int itemPosition) {
+			this.itemPosition = itemPosition;
+		}
+		public int getValue() {
+			return this.itemPosition;
+		}
+	}
+	private static Log logger = LogFactory.getLog(SyncmlServiceImpl.class);
+
+	/**
+	 * This method resolves the Syncml messages received through device and send the
+	 * response accordingly.
+	 *
+	 * @param request - Syncml requesy comes through the device
+	 * @return - Syncml response generated for the request
+	 */
+	@Override
+	public Response getInitialResponse(Document request)
```

Comment:
Keep the keyword "throws" within the first line itself. When you break the line from that particular point, it gives an indication that there are some exceptions following the method declaration.